reference/storage-class-parameters.md

```diff
@@ -51,4 +51,27 @@ Note:
 1. By default, the volumes are provisioned as `thick`. 
 2. For a pool of a particular size, say 10 Gigabytes, a volume > 10 Gigabytes cannot be created, as Mayastor 2.1.0 does not support pool expansion.
 3. The replicas for a given volume can be either all thick or all thin. Same volume cannot have a combination of thick and thin replicas
-{% endhint %}
\ No newline at end of file
+{% endhint %}
+
+
+## affinityGroup
+
+An `affinityGroup` represents a collection of volumes that belong to instances of Kubernetes StatefulSet. When a StatefulSet is deployed, each instance within the StatefulSet creates its own individual volume, which collectively forms the `affinityGroup`. Each volume within the `affinityGroup` corresponds to a pod of the StatefulSet.
+
+This feature enforces the following rules to ensure the proper placement and distribution of replicas and targets :
```

Comment:
`This feature enforces the following rules to ensure the proper placement and distribution of replicas and targets so that there isn't any single point of failure affecting multiple instances of statefulset.`

Reply:
Updated the text,